docs/src/components/Navbar.tsx

```diff
@@ -96,7 +97,7 @@ interface Props {
 }
 
 const Navbar = ({ location, docNavigation }: Props) => {
-  const [searchOpen, setSearchOpen] = React.useState<boolean>(false);
+  const { isSearchOpened, setSearchOpened } = useKeyboard();
```

Comment:
I recommend making `useKeyboard` behave similar to `useState` and returning array rather than an object, at least that's what I've been doing so far. Also, keep the same naming convention, i.e. `searchOpen` and `setSearchOpen` instead of `isSearchOpened` and `setSearchOpened`, just like `menuOpen` and `setMenuOpen` below.